src/index.js

```diff
@@ -5,14 +5,15 @@ var batch = require('./batch'),
     cache = require('./cache'),
     DatabaseAdapter = require('./DatabaseAdapter'),
     express = require('express'),
-    S3Adapter = require('./S3Adapter'),
     middlewares = require('./middlewares'),
     multer = require('multer'),
     Parse = require('parse/node').Parse,
     PromiseRouter = require('./PromiseRouter'),
     httpRequest = require('./httpRequest');
 
-import { default as GridStoreAdapter } from './GridStoreAdapter';
+import { default as GridStoreAdapter } from './Adapters/Files/GridStoreAdapter';
```

Comment:
you can just do `import GridStoreAdapter from './Adapters/Files/GridStoreAdapter';` if you are using default exports